### PR TITLE
fix a11y in dropdown

### PR DESCRIPTION
**Pull request by uzairr**
fix a11y related concerns in country dropdown.

VAN-559

### CODE REVIEW

File: src/redesign/register/CountryDropdown.jsx [hunk 1]
```diff
@@ -88,17 +88,9 @@ class CountryDropdown extends React.Component {
...
-    }
-    this.setState({
-      dropDownItems, icon: ExpandLess, errorMessage: '', showFieldError: false,
-    });
-
-    if (this.state.dropDownItems?.length > 0) {
-      this.setState({ dropDownItems: '', icon: ExpandMore });
+    const dropDownItems = this.getItems(e.target.value);
+    if (dropDownItems?.length > 1) {
+      this.setState({ dropDownItems, icon: this.expandMoreButton() });
```
Comment (waheedahmed) [inline src/redesign/register/CountryDropdown.jsx:93]: Need to set `errorMessage: ''` to clear the error message on `onFoucs`.

**Review by waheedahmed — COMMENTED** [review]
Also, we need to validate the value as well, currently we are just checking if the value is set which isn’t True with the typeahead search field because if the user enters an incorrect value; it does not show an error message on focus out.
Screenshot 2021-06-24 at 2 29 57 PM

File: src/redesign/_style.scss [hunk 2]
```diff
@@ -528,7 +528,7 @@ select.form-control {
 }
 
 .username-suggestion {
-  padding: 0.15rem 0.5rem;
```
Comment (waheedahmed) [inline src/redesign/_style.scss:531]: This is not the desired change, according to Figma design `padding-left/right` should be 2px and `font-size: 12px` and `line-height: 24px`.

Comment (uzairr) [reply src/redesign/_style.scss:531]: Screen Shot 2021-06-24 at 5 52 18 PM
I think this is not promissing with design ?

**Review by waheedahmed — APPROVED** [review]

**Pull request merged**